contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -266,6 +266,7 @@ BEGIN
 	CALL sys.babel_initialize_logins('sysadmin');
 	CALL sys.babel_create_builtin_dbs(sa_name);
 	CALL sys.initialize_babel_extras();
+	ANALYZE;
```

Comment:
Please add comment that we are analyzing all tables in the current postgres database.

Reply:
created a function to run analyze for objects in sys schema only.